commercetools-sdk-base/src/main/resources/META-INF/services/javax.money.spi.ServiceProvider

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2012, 2015, Credit Suisse (Anatole Tresch), Werner Keil and others by the @author tag.
```

Comment:
Do we need the copy right notice here? To me it seems unrelated to their code or am I wrong?